### PR TITLE
[PIN-3390] Catalog Management Cascading update

### PR DESCRIPTION
**Pull request by TonioGela**

### CODE REVIEW

File: project/Versions.scala [hunk 1]
```diff
@@ -17,12 +17,12 @@ object Versions {
 
 object PagopaVersions {
   lazy val commonsVersion                     = "1.0.x-SNAPSHOT"
-  lazy val agreementProcessVersion            = "1.0.x-SNAPSHOT"
-  lazy val catalogProcessVersion              = "1.0.x-SNAPSHOT"
+  lazy val agreementProcessVersion            = "PIN-3389-SNAPSHOT"
```
Comment (galales) [inline project/Versions.scala]: 🎺

File: src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/EServicesApiServiceImpl.scala [hunk 2]
```diff
@@ -330,22 +330,24 @@ final case class EServicesApiServiceImpl(
     requesterTenant <-
       if (requesterId != eService.producerId) tenantProcessService.getTenant(requesterId)
       else Future.successful(producerTenant)
-
     activeDescriptor = getActiveDescriptor(eService)
     latestAgreement <- agreementProcessService.getLatestAgreement(requesterId, eService)
+    hasCertifiedAttributes = activeDescriptor.fold(false)(descriptor =>
```
Comment (galales) [inline src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/EServicesApiServiceImpl.scala:335]: just to double check, is this the expected behaviour from the frontend?

Comment (TonioGela) [reply src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/EServicesApiServiceImpl.scala:335]: I'm pretty sure I've discussed it with @ruggerocastagnola, can you confirm?

Comment (TonioGela) [reply src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/EServicesApiServiceImpl.scala:335]: I just checked the logic with @ruggerocastagnola and it's ok

**Review by beetlecrunch — APPROVED** [review]

**Review by galales — APPROVED** [review]

**Pull request merged**